nx/source/services/hid.c

```diff
@@ -61,6 +76,28 @@ void hidExit(void)
     shmemClose(&g_hidSharedmem);
 }
 
+void hidReset(void) {
+    // Reset internal state
+    memset(&touchEntry, 0, sizeof(HIDTouchScreenEntry));
```

Comment:
You can turn all of this into an anonymous struct and then just have a single memset-call if you want to